guides/common/modules/proc_configuring-a-host-for-registration.adoc

```diff
@@ -1,44 +1,39 @@
 [id="Configuring_a_Host_for_Registration_{context}"]
 = Configuring a Host for Registration
 
-{RHEL} virtual machines register to Customer Portal Subscription Management by default.
-You must update each virtual machine configuration so that they receive updates from the correct {Project} Server or {SmartProxyServer}.
+Configure your host for registration to {ProjectServer} or {SmartProxyServer}.
 
 .Prerequisites
-* Hosts must be using the following {EL} version:
-** 6.4 or later
-** 7.0 or later
-* All architectures of {EL} are supported (i386, x86_64, s390x, ppc_64).
-* Ensure that a time synchronization tool is enabled and runs on the {ProjectServer}s, any {SmartProxyServers}, and the hosts.
-** For {EL} 6:
-+
-----
-# chkconfig ntpd on; service ntpd start
-----
-** For {EL} 7:
+* The host must be using a supported OS version.
+For more information, see xref:supported-clients_{context}[].
+* Ensure that a time-synchronization tool is enabled and runs on {ProjectServer} and any {SmartProxyServers}.
+
+.Procedure
+. Ensure that a time-synchronization tool is enabled and running on the host.
+ifdef::client-content-dnf[]
+** For {EL} 7 and newer:
 +
 ----
 # systemctl enable --now chronyd
 ----
-* Ensure that the daemon `rhsmcertd` is enabled and running on the hosts.
 ** For {EL} 6:
 +
 ----
-# chkconfig rhsmcertd on; service rhsmcertd start
+# chkconfig ntpd on
+# service ntpd start
 ----
-** For {EL} 7:
+endif::[]
+. Ensure that the `rhsmcertd` daemon is enabled and running on the host.
```

Comment:
I wonder why this is needed. Perhaps it's only for RHEL? @stejskalleos any idea?

Reply:
So do I, we also mention it in the global registration prerequisites.

Reply:
Can it be an additional requirement for non-RHEL systems?

Reply:
No, rhsmcertd is Red Hat Subscription Manager Certificate Daemon. I'd say it's **only** needed on Katello when registering a content machine. But if it's needed, then the registration script should ensure it's running. It's poor UX when you need to manually execute steps when a script can do it.

Syncing time is different, because the time needs to be (reasonably) in sync for HTTPS to work. If it's off, it may think a certificate is not yet valid or already expired while it's supposed to still be valid. Since the script is retrieved via HTTPS, you have a chicken-and-egg situation.

Reply:
TBH I've never come to this problem when registering RHEL7 machines.
What happens if it is not installed and enabled before the registration?

Reply:
@ekohl @stejskalleos Unless you intend to resolve this conversation quickly, I'm leaving it for future generations to resolve. :)

Reply:
23ce7ba762e2346958daccd989071c6e390ba4aa added it, but that was before global registration. In b88b60bff56e067a7c62057a0010719a1d541cdf it was copied into the global registration instructions by @stejskalleos but if you now say that it's no longer needed then I'd be tempted to remove the instructions.

Reply:
Yop let's remove the instructions